src/main/java/com/amihaiemil/docker/Container.java

```diff
@@ -151,5 +151,18 @@ void remove(final boolean volumes, final boolean force, final boolean link)
      *  the expected one (204 NO CONTENT).
      */
     void unpause() throws IOException;
-    
+
+
+    /**
+     * Waits on this container.
+     * @param state The state to wait for. One of "not-running"
+     * (the default if null), "next-exit", or "removed"
+     * @see <a href="https://docs.docker.com/engine/api/v1.35/#operation/ContainerWait">
+     * Wait Container</a>.
+     * @throws IOException If something goes wrong.
+     *  the expected one (200).
```

Comment:
@ry99 I think this javadoc is broken :D

Reply:
Javadoc passes the checkstyle rules now. sorry